bp_common/src/include/bp_common_me_if.vh

```diff
@@ -344,14 +312,12 @@ typedef enum bit [1:0]
  * e_lce_resp_null_wb indicates that the LCE never wrote to the cache block and the block is still
  *   clean. The data field should be 0 and is invalid.
  */
-
 typedef enum logic
 {
-  e_lce_resp_wb              = 1'b0  // Normal Writeback Response (full data)
-  ,e_lce_resp_null_wb        = 1'b1  // Null Writeback Response (no data)
-} bp_lce_cce_resp_msg_type_e;
+  e_lce_cce_resp_wb              = 1'b0  // Normal Writeback Response (full data)
+  ,e_lce_cce_resp_null_wb        = 1'b1  // Null Writeback Response (no data)
+} bp_lce_cce_resp_data_msg_type_e;
```

Comment:
Can we collapse this into the resp_msg_type?  It won't require any additional bits, but I think it's confusing to have two msg type fields in one packet

Reply:
Same comment on cmd_msg_type

Reply:
agreed. I was just coming to the same conclusion while updating the CCE implementation.